### DISCUSSION TITLE
Make an object disapear (visible to invisible)

### DISCUSSION THREAD

**Discussion by Airioo** [bnpr/Malt#220] [Q&A] [answered]
Hello ! Happy New Year all :) !
I'm wondering if it's possible to make an object disapear (visible to invisible) ? 
Thanks a lot !

**Comment by pragma37** [comment, marked as answer]
Hi! Happy new year!
You just have to change the alpha value in PixelOutput > color.

**Reply by Airioo** [reply]
Where is it please ?

**Reply by pragma37** [reply]
In the *PixelOutput* node from your material. (Or directly in *COMMON_PIXEL_SHADER Output* if you are using the Development version)
You must also have the *Transparency* setting enabled in the *Material* settings panel.

**Reply by Airioo** [reply]
Ok thanks I'll look for it !

**Reply by Airioo** [reply]
That's work fine thanks a lot !
